modules/javafx.controls/src/main/java/javafx/scene/control/skin/TableColumnHeader.java

```diff
@@ -48,10 +50,7 @@
 import javafx.scene.AccessibleAttribute;
 import javafx.scene.AccessibleRole;
 import javafx.scene.Node;
-import javafx.scene.control.ContextMenu;
-import javafx.scene.control.Label;
-import javafx.scene.control.TableColumn;
-import javafx.scene.control.TableColumnBase;
+import javafx.scene.control.*;
```

Comment:
Wildcard imports should be avoided. Please add specific imports that you need.

Reply:
Thanks I have not seen that, my IDE was misconfigured